source/help/messaging/executing-commands.md

```diff
@@ -0,0 +1,18 @@
+# Executing Commands
+___
+Slash commands allow users to perform operations in Mattermost or interact with external applications by typing into the text input box. A user will enter a `/` followed by a command and some arguments to perform actions. There are built-in slash commands that come with all Mattermost installations, and optionally, custom slash commands may be configured to interact with external applications. To learn more about configuring custom slash commands, please visit the [developer slash command documentation page](../developer/slash-commands.md).
+
+## Built-in Commands
+
+The following slash commands are available on all Mattermost installations:
+
+![commands](../../images/slashCommandsTable.PNG)
+
+Begin by typing `/` and a list of slash command options appears above the text input box. The autocomplete suggestions help by providing a format example in black text and a short description of the slash command in grey text.
+
+![autocomplete](../../images/slashCommandsAutocomplete.PNG)
```

Comment:
Cropping of this image seems imprecise--Help in bottom right seems too close to edge, also left and right margins and top and bottom margins don't seem even. 
![image](https://cloud.githubusercontent.com/assets/177788/13417145/ec21ef32-df1d-11e5-9754-92bbf50a756e.png)

We have an [FX guideline around even spacing](http://docs.mattermost.com/fx-guidelines/ui-elements.html#alignment-of-elements) propose applying it in docs as well?